windows-driver-docs-pr/devtest/28726-banned-api-usage-use-updated-function-replacement.md

```diff
@@ -9,11 +9,56 @@ f1_keywords:
 # C28726
 
 
-warning C28726: Banned API Usage
+**warning C28726: Banned API Usage L2**\
+_Banned API Usage:  *function name* is a Banned API as listed in dontuse.h for security purposes._
 
-This warning indicates that a function is being used that has been banned or deprecated, and has a more robust and secure replacement.
-
-To find the replacement, search for the name of the deprecated function in the [Windows Driver Development](../index.yml) documentation. The documentation lists the name of the deprecated function and suggests replacements.
-
- 
+This warning indicates that a function is being used that has been banned and has a more robust or secure replacement. This specific error indicates that the banned function has potential to overflow a buffer. A list of all banned functions covered by this error, why they are banned, and recommended replacements can be found after the following example: 
+ ## Example 
 
+The following code generates this warning: 
+```
```

Comment:
I think you can get markdown to do syntax highlighting by specifying the language here.

Something like this maybe?
```suggestion
```c
```

Reply:
Added cpp to this PR, will add to others now!